### PR TITLE
Added Terraform version

### PR DESCRIPTION
**Pull request by eyalatzmon**
Added Terraform version to template 
set default terraform version =  0.15.1
added to data and resource 
updated template test with tf version

### CODE REVIEW

File: client/model.go [hunk 1]
```diff
@@ -125,6 +126,7 @@ type Template struct {
 	Type                 string           `json:"type"`
 	GithubInstallationId int              `json:"githubInstallationId"`
 	UpdatedAt            string           `json:"updatedAt"`
+	TerraformVersion	 string			  `json:"terraformVersion"`
```
Comment (eyalatzmon) [inline client/model.go:129]: the indentation here looks strange, but in VS it looks fine.

**Review by roni-frantchi — APPROVED** [review]
LGTM

File: client/model.go (hunk 1 above)
Comment (eranelbaz) [reply client/model.go:129]: Move to webstorm

**Comment by eyalatzmon** [comment]
bors merge

**Comment by eranelbaz** [comment]
@eyalatzmon note we don't have bors on this repo
so just press the merge button

**Pull request merged**